src/lv_conf_zephyr.h

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Sorry for the confusion in the previous discussion but I would not add this file to LVGL and leave it to zephyr to provide it.